docker-compose.full.yml

```diff
@@ -12,42 +12,6 @@ services:
     extends:
       file: docker-services.yml
       service: lb
-    links:
-      - frontend
-  # Frontend
-  frontend:
```

Comment:
`frontend` is still needed, as a reverse proxy in front of `uwsgi`. You can remove the `react_static` volume and link to it